spec.md

```diff
@@ -137,6 +137,28 @@ attributes and the event data will be materialized. For example, in the case
 of a JSON serialization, the context attributes and the event data might
 both appear within the same JSON object.
```

Comment:
Suggest to add something like the following to the "Overview" section or another appropriate place: 
"If a metadata in the format of a property bag will be used by a large number of use cases, then a consistent name will be given to the property bag and its semantics should be defined in the main spec.md "

Rationale for this: 
•A consistent name is needed to prevent different event producers from giving different/random property names resulting in many property names for the same property bag.
•Simplify the Event consumers’ implementation. Instead of having to parse different property names from different event producers for the same type of information, the consumer just needs to parse one name.
•There are other standard protocols that define the names for property bags

Reply:
I added some more text to the primer, at the end of the extension section, to address this.